### PR TITLE
Vector Tile Layer

### PR DESCRIPTION
**Pull request by Nour-Cheour10**

### CODE REVIEW

**Comment by Nour-Cheour10** [comment]
update galata references

**Comment by Nour-Cheour10** [comment]
Bot update galata references

**Comment by Nour-Cheour10** [comment]
please update snapshots

**Comment by martinRenou** [comment]
please update snapshots!!!

**Comment by martinRenou** [comment]
This is conflicting with the main branch, you'd need to rebase

**Comment by Nour-Cheour10** [comment]
please update snapshots

**Comment by martinRenou** [comment]
Triggering CI

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4 || ^5 || ^6",
+    "dotenv": "^16.4.5",
```
Comment (martinRenou) [inline package.json]: Do you need this?

**Comment by Nour-Cheour10** [comment]
please update snapshots

**Comment by martinRenou** [comment]
Triggering CI

File: package.json (hunk 1 above)
Comment (Nour-Cheour10) [reply package.json]: no I'll take it off

**Comment by Nour-Cheour10** [comment]
please update snapshots

**Comment by Nour-Cheour10** [comment]
please update snapshots

**Comment by martinRenou** [comment]
Triggering CI

File: examples/example-data.json
Comment (martinRenou) [inline examples/example-data.json:1]: This file is quite big, and I don't see it being used in your notebooks, should we remove them from the repository?

**Review by martinRenou — CHANGES_REQUESTED** [review]

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**